components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/internal/database/DashboardMetadataDao.java

```diff
@@ -58,6 +57,61 @@ public DashboardMetadataDao(DataSource dataSource, QueryManager queryManager) {
         this.queryManager = queryManager;
     }
 
+    public void initDashboardTable() throws DashboardException {
+        if (!tableExists(QueryManager.DASHBOARD_RESOURCE_TABLE)) {
+            this.createDashboardResourceTable();
+        }
+    }
+
+    /**
+     * Create dashboard resource table.
```

Comment:
missing `@throws` in this java doc.

Reply:
Fixed in: b39054da2deccdfef8422d6423f92ba62064dd34